### ISSUE TITLE
Oldstyle and symbol use - use "symbols" instead of only "marvosym"

### ISSUE THREAD

**Issue by LLdaniel** [moderncv/moderncv#208]
**Describe the bug**
Oldstyle uses in contrast to all other styles the following icon selection:
https://github.com/moderncv/moderncv/blob/cdf027a9da1f16038c2df3c02147b3b316741dac/moderncvstyleoldstyle.sty#L48-L57

**Expected Behavior**
Due to symmetry reasons wie should also use `\moderncvicons{symbols}` for oldsystyle as well. 

**Minimum Working Example to Reproduce the Problem**
Please use the `template.tex` with "oldstyle".

**Comment by thecaligarmo** [comment]
Should we get rid of the "if xeluatex use Awesome" part as well to match the other ones or just change mavsym?

**Comment by LLdaniel** [comment]
IMO we could remove the case distinction with _xeluatex_, because everything else also does not differentiate between them in the style file. It was introduced in 3513820. 
```bash
└─▶ $ grep -r "moderncvicons{"
moderncvstyleclassic.sty:    \moderncvicons{symbols}     % use symbols
moderncvstyleclassic.sty:    \moderncvicons{letters}     % do not use symbols, use letters
moderncvstylefancy.sty:    \moderncvicons{symbols}     % use symbols
moderncvstylefancy.sty:    \moderncvicons{letters}     % do not use symbols, use letters
CHANGELOG:  (can be changed back with \moderncvicons{marvosym}).
CHANGELOG:  \moderncvicons{<iconsetname>}, where <iconsetname> can be "marvosym" (used by
manual/moderncv_userguide.tex:  \moderncvicons{<symbols|letters>}
moderncvstylecasual.sty:    \moderncvicons{symbols}     % use symbols
moderncvstylecasual.sty:    \moderncvicons{letters}     % do not use symbols, use letters
moderncvstylebanking.sty:    \moderncvicons{symbols}     % use symbols
moderncvstylebanking.sty:    \moderncvicons{letters}     % do not use symbols, use letters
moderncv.cls:% usage: \moderncvicons{<icon set name>}
moderncvstyleempty.sty:    \moderncvicons{symbols}     % use symbols
moderncvstyleempty.sty:    \moderncvicons{letters}     % do not use symbols, use letters
moderncvstylecontemporary.sty:\moderncvicons{symbols}
moderncvstyleoldstyle.sty:    \moderncvicons{awesome}     % use font-awesome symbols if lualatex or xelatex is used
moderncvstyleoldstyle.sty:    \moderncvicons{marvosym}    % use marvosym symbols pdflatex is used
moderncvstyleoldstyle.sty:    \moderncvicons{letters}    % do not use symbols, use letters
```
I also tried to compile it with "symbols" and `pdflatex` which also worked except some "academicons". The reason is `pdflatex`, see here https://github.com/moderncv/moderncv/blob/cdf027a9da1f16038c2df3c02147b3b316741dac/moderncviconssymbols.sty#L16-L26

tl;dr: yes

**Comment by thecaligarmo** [comment]
For the "academicons", this happens on other styles as well, so that seems like a more global bug so I didn't include it (I tried it with "banking" and I got the same error)
